src/main/java/de/metas/ui/web/product/process/WEBUI_M_Product_LotNumber_Lock.java

```diff
@@ -148,6 +155,18 @@ private void createQuarantineHUsByLotNoLockId(final int lotNoLockId)
 		}
 
 	}
+	
+	private boolean existsDDOrderLineCandidateForHUId(final int huId)
```

Comment:
pls move it to IHUDDOrderDAO.
No database operations outside Repository/DAO are allowed.

Reply:
also pls declare your services at the top of the process.

Reply:
also check above, don't use `load` outside of Repository/DAO